src/testing/test-utils.ts

```diff
@@ -1,5 +1,39 @@
-import { act, renderHook, RenderHookOptions, RenderHookResult, within } from '@testing-library/react';
+import { Theme, ThemeProvider } from '@terra-ui-packages/components';
+import {
+  act,
+  render,
+  renderHook,
+  RenderHookOptions,
+  RenderHookResult,
+  RenderOptions,
+  within,
+} from '@testing-library/react';
 import userEvent from '@testing-library/user-event';
+import { PropsWithChildren, ReactElement } from 'react';
+import { h } from 'react-hyperscript-helpers';
+
+const testTheme: Theme = {
```

Comment:
feels like this should move over to a test-utils within components package... I don't want these utils to just become new dumping grounds.

Reply:
Components' test-utils is not exposed to consumers. It can't be included in the package build because it relies on dev dependencies.

A possible natural value for this is the default Terra theme from brands.

But generally, I agree that this module's growth is starting to run amok. I think it can be addressed with a combination of moving stuff to the test-utils package and breaking this up into multiple modules.

Reply:
yeah, ignore the urgency on this one.... thought this was in package test-utils... it;s fine where it is for now.